howso/utilities/installation_verification.py

```diff
@@ -67,17 +67,17 @@ class Status(IntEnum):
     OK = 4
 
 
-Requirements = Iterable[Union[type, object]]
+Requirements = t.Iterable[type | object]
```

Comment:
This case should continue to use `Union`, not the `|` form.

Reason, the `from __future__ import annotations` import at the top will allow the `|` to work on older Python's (3.9, in particular), but only on function, class, or method signatures, it won't help here where we're defining a custom Type (as this is not a function, method or class signature).

Reply:
So only use | in place of Union for functions, methods or class signatures?

Reply:
> So only use | in place of Union for functions, methods or class signatures?

Yes. Not in custom type declarations like this is here.